ohif-configurable/config-mode/src/isValidMode.ts

```diff
@@ -0,0 +1,15 @@
+/** Any modality but SM or ECG */
```

Comment:
This is the only change required here, but the other items are useful generally.